### ISSUE TITLE
[release-1.6] pull-e2e-gce is broken

### ISSUE THREAD

**Issue by enisoc** [kubernetes/test-infra#5280]
For pulls on the 1.6 branch, the apiserver never comes up, failing with this error:

```
error in initializing storage factory: group version admissionregistration.k8s.io/v1alpha1 that has not been registered
```

I think this is caused by https://github.com/kubernetes/test-infra/pull/5077

cc @krzyzacy @caesarxuchao

**Comment by krzyzacy** [comment]
ouch
/assign

**Comment by BenTheElder** [comment]
/assign
Also going to add comments to the config to explain how to do this correctly next time.

**Comment by BenTheElder** [comment]
this should have been fixed in #5281

**Comment by BenTheElder** [comment]
The job appears to have passed in #46450
